### DISCUSSION TITLE
How do you actually... get to the new dimension?

### DISCUSSION THREAD

**Discussion by doctorduck2000** [syldium/worldgen#36] [Q&A]
Can you set up a custom portal or do you need to use commands or what?

**Comment by syldium** [comment]
You can teleport the player to any dimension using the `execute` command :
```mcfunction
execute in test:my_custom_dimension run tp @s ~ ~ ~
```
If a dimension is not listed when typing the command, then it most likely failed to load.

Creating custom portals requires more work since the vanilla system is currently not exposed for datapacks. Some mods such as [Immersive Portals](https://www.curseforge.com/minecraft/mc-mods/immersive-portals-mod) or [Datapack Portals](https://www.curseforge.com/minecraft/mc-mods/datapack-portals) allow datapacks to hook into their portal system.

Finally, there is the vanilla method of recreating the portal detection mechanism and other elements in a datapack. There are some videos around the subject.

**Reply by doctorduck2000** [reply]
Thank you!
